.gitignore

```diff
@@ -187,7 +187,7 @@ fabric.properties
 
 .idea/*
 
-!.idea/codeStyles
+#!.idea/codeStyles
```

Comment:
Are we removing this from the .gitignore?

Reply:
Yea, I'm not sure why that was excluded. The `.idea` directory is phpstorm related. I ended up with a codeStyles config file that was committed so making sure we don't get those in here by accident.